tests/test_config_consistency.py

```diff
@@ -662,13 +661,28 @@ def get_runtime_metrics(agent):
     return runtime_metrics_gauges, runtime_metrics_sketches
 
 
-# Parse the JSON-formatted log message from stdout and return the 'dd' object
+# Parses the JSON-formatted log message from stdout and returns it
 def parse_log_injection_message(log_message):
     for data in stdout.get_data():
         try:
             message = json.loads(data.get("message"))
         except json.JSONDecodeError:
             continue
-        if message.get("dd") and message.get(log_injection_fields[context.library.library]["message"]) == log_message:
+        # APMAPI-1199
+        if (
+            context.library == "nodejs"
+            and message.get(log_injection_fields[context.library.library]["message"]) == log_message
```

Comment:
This condition is hard to understand. Can we leave comment to document why we need to to check log_injection_fields for the nodejs weblog app. It would be nice if we could avoid this but we shouldn't invest too much time into it if this is challenging.